book/1-pre-project-phase/business_challenge_2_ai_problem.md

```diff
@@ -0,0 +1,42 @@
+# How does the AI engineer translate the business challenge into an AI problem?
+Contributor: Tan Kwan Chet 
```

Comment:
General comment: do we want to put the title/team of the person? Eg. ```Contributor: Tan Kwan Chet, Lead AI Technical Consultant```. This may help readers better understand the role and expertise of the contributors.